### DISCUSSION TITLE
Can some people join my little prodigy class I made!

### DISCUSSION THREAD

**Discussion by MisakeTheRat** [Prodigy-Hacking/ProdigyMathGameHacking#1241] [(A) Anything]
I just want a few prodigy friends to battle with the code is *******

**Comment by afkvido** [comment]
> I just want a few prodigy friends to battle with the code is [-------]

Good evening @MisakeTheRat,

Sadly, your post is not allowed I think. Students in a class can have their password, grade, lessons, and more changed by teachers, that's why random classes are seen as malicious.

**Reply by MisakeTheRat** [reply]
Oh I don't own the class my mother does.

**Reply by MisakeTheRat** [reply]
Sorry tho

**Reply by afkvido** [reply]
> Oh I don't own the class my mother does.

**LMAO**

**Reply by afkvido** [reply]
> Sorry tho

it's ok, can you please censor the class code? thanks!

**Reply by MisakeTheRat** [reply]
> > Oh I don't own the class my mother does.
> 
> **LMAO**

:,   )

**Reply by MisakeTheRat** [reply]
> > Sorry tho
> 
> it's ok, can you please censor the class code? thanks!

Ok

**Reply by afkvido** [reply]
Thanks! Have a good day!

**Comment by donaldli2020** [comment]
If you don't want to share personal info, add some "Friends."

**Reply by afkvido** [reply]
"Get some friends"

**Reply by MisakeTheRat** [reply]
I said prodigy friends, nobody really plays prodigy, and how is sharing a code for a game personal info?? That comment makes no sense

**Reply by MisakeTheRat** [reply]
The reason why I looked on here is because prodigy chat is limited and you get ignored

**Reply by MisakeTheRat** [reply]
I already have 
lots of friends in person lmao
> "Get some friends"

**Reply by afkvido** [reply]
> I said prodigy friends, nobody really plays prodigy, and how is sharing a code for a game personal info?? That comment makes no sense

i was JOKING

**Reply by afkvido** [reply]
> The reason why I looked on here is because prodigy chat is limited and you get ignored
> 
> 

I understand :)
